environment-dev.yml

```diff
@@ -11,8 +11,8 @@ dependencies:
   - cftime
   - cf_xarray >=0.7.6
   - clisops >=0.15
-  - dask >=2024.8.1,<2024.12  # FIXME: Remove upper pin when https://github.com/pangeo-data/rechunker/pull/156 is merged
-  - flox !=0.9.14  # FIXME: 0.9.14 is a broken version. This pin could be removed eventually.
+  - dask >=2024.8.1
+  - flox
```

Comment:
Would we want to set a base version here?

Reply:
`0.9.14` was an issue, but otherwise I don't think that we do.